symengine/tests/basic/test_relationals.cpp

```diff
@@ -100,6 +102,21 @@ TEST_CASE("Comparing Relationals", "[Relationals]")
     a = Lt(x, y);
     b = Lt(x, y);
     CHECK(eq(*a, *b));
+
+    RCP<const Basic> im1 = integer(-1);
+    RCP<const Basic> i0 = integer(0);
+    RCP<const Basic> i1 = integer(1);
+    RCP<const Basic> r1 = complex_double(std::complex<double>(0.1, 0.2));
+    RCP<const Basic> r2 = complex_double(std::complex<double>(1, 0.2));
+    RCP<const Basic> comp = integer(r1->compare(*r1));
+    CHECK(eq(*comp, *i0));
+    comp = integer(r1->compare(*r2));
+    CHECK(eq(*comp, *im1));
+    comp = integer(r2->compare(*r1));
+    CHECK(eq(*comp, *i1));
+    r2 = complex_double(std::complex<double>(0.1, 0.3));
+    comp = integer(r2->compare(*r1));
+    CHECK(eq(*comp, *i1));
```

Comment:
Since this is testing a method of the ComplexDouble class I would move this to test_number.cpp or create a new test_complex.cpp file.

Reply:
Ok, I've added a comparison test case in test_number.cpp